src/main/resources/templates/pps/confirmationPage.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 
 <html xmlns:th="http://www.thymeleaf.org"
-      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" xmlns="http://www.w3.org/1999/html"
```

Comment:
What's the reason for changing this when it looks like none of the other templates have this extra namespace?

Reply:
Removed, I think it is because I was adding that for some style when developing